src/pybamm/experiment/step/base_step.py

```diff
@@ -55,6 +55,9 @@ class BaseStep:
         A description of the step.
     direction : str, optional
         The direction of the step, e.g. "Charge" or "Discharge" or "Rest".
+    skip_ok : bool, optional
+        If True, the step will be skipped if it is infeasible at the initial conditions.
+        Default is False.
```

Comment:
Do we actually need this option or can it just always be True?

Reply:
Ignore where the comment is - meant this as more of a general comment

Reply:
my instinct is to not change things unless its very obvious, but if you think we should eliminate it or make it default true that would be fine with me.

Reply:
let's do default true